### ISSUE TITLE
Auto Proficiency causing errors when adding Bard musical instruments

### ISSUE THREAD

**Issue by emirkmo** [trioderegion/dnd5e-helpers#57]
Specifically, I am using DDB-Importer by Mr. Primate and dnd5e-helpers, as well as DAE and SRD-DAE compendia. The issue comes about when importing a bard character with proficiency in musical instruments such as drums.

I get a pop-up telling me to manually check to proficiency but I also get some console errors. It's not clear to me what it means to check the proficiency manually?

Here is one of the errors from the console:
```
foundry.js:2499 ReferenceError: pass_type is not defined
    at AutoProfArmor_createOwnedItem (dnd5e-helpers.js:716)
    at dnd5e-helpers.js:1142
    at Function._call (foundry.js:2496)
    at Function.callAll (foundry.js:2456)
    at Function._handleCreateEmbeddedEntity (foundry.js:29908)
    at Actor5e.createEmbeddedEntity (foundry.js:29879)
    at async Actor5e.createEmbeddedEntity (foundry.js:31075)
    at async migrateActorDAESRD (main.js:15987)
    at async CharacterImport.parseCharacterData (main.js:18005)
    at async HTMLButtonElement.<anonymous> (main.js:17651)
_call @ foundry.js:2499
callAll @ foundry.js:2456
_handleCreateEmbeddedEntity @ foundry.js:29908
createEmbeddedEntity @ foundry.js:29879
```

This one is specifically happening during the creation of 1 of 3 instruments. It happens for all three.

```
foundry.js:2499 ReferenceError: pass_type is not defined
    at AutoProfArmor_createOwnedItem (dnd5e-helpers.js:716)
    at dnd5e-helpers.js:1142
    at Function._call (foundry.js:2496)
    at Function.callAll (foundry.js:2456)
    at Function._handleCreateEmbeddedEntity (foundry.js:29908)
    at Actor5e.createEmbeddedEntity (foundry.js:29879)
    at async Actor5e.createEmbeddedEntity (foundry.js:31075)
    at async CharacterImport.importCharacterItems (main.js:17783)
    at async CharacterImport.processCharacterItems (main.js:17944)
    at async CharacterImport.parseCharacterData (main.js:17983)
_call @ foundry.js:2499
callAll @ foundry.js:2456
_handleCreateEmbeddedEntity @ foundry.js:29908
createEmbeddedEntity @ foundry.js:29879
async function (async)
createEmbeddedEntity @ foundry.js:29869
createEmbeddedEntity @ foundry.js:31075
createEmbeddedEntity @ entity.js:541
(anonymous) @ main.js:17380
importItems @ main.js:17379
importCharacterItems @ main.js:17783
async function (async)
importCharacterItems @ main.js:17781
processCharacterItems @ main.js:17944
async function (async)
processCharacterItems @ main.js:17869
parseCharacterData @ main.js:17983
async function (async)
parseCharacterData @ main.js:17963
(anonymous) @ main.js:17651
async function (async)
(anonymous) @ main.js:17643
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
```
```
ReferenceError: pass_type is not defined
    at AutoProfArmor_createOwnedItem (dnd5e-helpers.js:716)
    at dnd5e-helpers.js:1142
    at Function._call (foundry.js:2496)
    at Function.callAll (foundry.js:2456)
    at Function._handleCreateEmbeddedEntity (foundry.js:29908)
    at Actor5e.createEmbeddedEntity (foundry.js:29879)
    at async Actor5e.createEmbeddedEntity (foundry.js:31075)
    at async migrateActorDAESRD (main.js:15987)
    at async CharacterImport.parseCharacterData (main.js:18005)
    at async HTMLButtonElement.<anonymous> (main.js:17651)
_call @ foundry.js:2499
callAll @ foundry.js:2456
_handleCreateEmbeddedEntity @ foundry.js:29908
createEmbeddedEntity @ foundry.js:29879
async function (async)
createEmbeddedEntity @ foundry.js:29869
createEmbeddedEntity @ foundry.js:31075
createEmbeddedEntity @ entity.js:541
createOwnedItem @ foundry.js:31211
migrateActorDAESRD @ main.js:15987
async function (async)
migrateActorDAESRD @ main.js:15985
parseCharacterData @ main.js:18005
async function (async)
parseCharacterData @ main.js:17963
(anonymous) @ main.js:17651
async function (async)
(anonymous) @ main.js:17643
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
```

**Comment by trioderegion** [comment]
There is a lingering error related to that symbol ``pass_type``. Opening back up this ticket so that we can trace it. Thanks for the report and stack trace :)

**Comment by trioderegion** [comment]
Fixed in 1.8.8
